stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -54,17 +54,16 @@ public func enableTPU(serverAddress: String? = nil, infeed: Bool = true) {
 // with enableTPU() above.
 @_transparent
 public func enableGPU() {
-  _RuntimeConfig.executionMode = .gpu
   #tfop("tfc.configureGPU") as Void
 }
 
 @_frozen
 public enum _ExecutionMode : Equatable {
-  /// Classical TF interpreter backend, on CPU.
-  case cpu
-  /// Classical TF interpreter backend, on GPU.
-  case gpu
-  /// TPU backend.
+  /// CPU or GPU execution.
+  case auto
```

Comment:
If `cpu` and `gpu` execution modes are merged, is it still possible to run CPU tests if Swift is built with `-DCUDA`?

Reply:
Yes! With the new device placement design, if user does not specify `-Xllvm -tf-target-gpu`, the graph runs on CPU. In other words, we are not relying on TF-level soft placement anymore. Our generated graph ops always have explicit device placement.